app/lib/methods/helpers/image.ts

```diff
@@ -0,0 +1,10 @@
+import { Image } from 'react-native';
+
+export const isImageURL = async (url: string) => {
+	try {
+		const result = await Image.prefetch(url);
```

Comment:
We can't do this, because prefetch downloads the image locally to be used later.
You're just using the return to understand if it's an image or not.
That's wrong, because we would be always persisting images on `RN's Image` lib to be rendered on `fast-image`. We would download the same image twice.
Also, since this persist happens on `RN's Image`, `clear cache` won't be able to clean it.